rllib/algorithms/algorithm_config.py

```diff
@@ -1993,14 +2021,39 @@ def debugging(
 
         return self
 
+    @ExperimentalAPI
+    def rl_module(
```

Comment:
From now on, the algo_config changes related to only rl_module can be contained in this experimental api